WalletWasabi/Blockchain/Transactions/TransactionFactory.cs

```diff
@@ -65,15 +65,16 @@ public TransactionFactory(Network network, KeyManager keyManager, ICoinsView coi
 		}
 
 		// Get allowed coins to spend.
-		var availableCoinsView = Coins.Unspent();
+		var availableCoinsView = Coins.Available().Unspent();
```

Comment:
As far as I can remember the original concept was to only exclude coins from those in a CJ **and in a critical section**.

Reply:
`SmartCoin` and the `CoinsView` only know about the `CoinJoinInProgress` boolean property.
It cannot distinguish between CJ phases.